### PR TITLE
perfecto-upload 0.0.6

### PR DESCRIPTION
**Pull request by sreevatsa-ramya**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] __I will not move an already shared step version's tag to another commit__
- [ ] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ ] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ ] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/perfecto-upload/0.0.6/step.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+summary: |
+  Upload APK/IPA files to Perfecto cloud repository
+description: |
+  Upload your build files to Perfecto for remote device testing. Upload your IPA/APK file to Perfecto repository
+website: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload
+source_code_url: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload
+support_url: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload/issues
+published_at: 2020-06-27T19:13:46.420922+05:30
+source:
+  git: https://sreevatsa-ramya:Sreesmrithi123%24@github.com/PerfectoCode/Bitrise_Perfecto_FileUpload.git
```
Comment (koral--) [inline steps/perfecto-upload/0.0.6/step.yml:11]: Credentials shouldn't be there.

**Comment by sreevatsa-ramya** [comment]
Hi,

I am unable to see these credentials in step.yml file -
https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload/blob/master/step.yml.
Please let me know where to find it

Sree

File: steps/perfecto-upload/0.0.6/step.yml (hunk 1 above)
Comment (lpusok) [reply steps/perfecto-upload/0.0.6/step.yml:11]: It is possible that it was specified in this format for the checked out git repository, and got inserted in this format.
Will check if it is possible to filter out from Bitrise tooling side. Until now pleas check the git remote URL in your local checked out repo.

**Comment by trapacska** [comment]
@sreevatsa-ramya it is not in your step.yml. The sharing process creates this based on your local git config. To modify it you have to edit either this PR (here: https://github.com/PerfectoCode/bitrise-steplib/tree/perfecto-upload-0.0.6) or you have to fix your global git config to not to include these credentials

**Comment by sreevatsa-ramya** [comment]
Hi,

I have updated step.ym file for 0.0.6. Please check

Best Regards,
Sreevatsa

**Pull request merged**